MANIFEST.in

```diff
@@ -9,5 +9,23 @@ include RELEASE.rst
 include setup.py
 include setup.cfg
 
-recursive-include astropy_helpers *
-exclude astropy_helpers/.*
+# Include the docs and examples
+recursive-include docs *
+recursive-include examples *
+
```

Comment:
```suggestion

prune build
prune docs/_build
prune docs/api

```
Just in case?

Reply:
hehe you noticed that at the same time I did.